mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.tmpl

```diff
@@ -2280,6 +2302,24 @@ func databaseVersionDiffSuppress(_, oldVersion, newVersion string, _ *schema.Res
 	return false
 }
 
+func availabilityTypeDiffSuppress(_, old, new string, d *schema.ResourceData) bool {
```

Comment:
could you give a brief description about why the diff suppress funcs are necessary?